build.gradle

```diff
@@ -13,38 +12,49 @@ sourceCompatibility = JavaVersion.VERSION_11  // source-code version and must be
 targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
 
 repositories {
-    mavenCentral()
+    jcenter()
+    maven { url 'https://jitpack.io' }
 }
 
+def junit5Version = '5.6.0'
+def zalandoVersion = '0.23.0'
+
 dependencies {
+    implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: 'v0.3.0'
+
     implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.10'
     implementation group: 'commons-io', name: 'commons-io', version: '2.6'
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.9'
-    implementation group: 'javax.ws.rs', name: 'javax.ws.rs-api', version: '2.1.1'
     implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
```

Comment:
Can use Jackson instead of Gson now to remove a dependency and save some space. Jackson is included in the Lambda SDK already.

Reply:
True